### PR TITLE
Add svg-logo, change logo.js to process svg-images

### PR DESCRIPTION
**Pull request by Phil-Ah**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,15 +5,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
 ### Changed
 - update dependencies in package,json
 - changed color values of various css elements to enhance accessibility
 - recolor of navbar-inverse
+- include svg-logo in index
```
Comment (ppxl) [inline CHANGELOG.md:12]: Missing reference, wording:

Missing reference:
It is hard to backtrack changes to their requirements when no links or references are given. If I get it right you base this PR on issue #16 (whose description could be more tacky as well, but one step after another).

Wording:
I miss in this changelog what actually depicts the logo being replaced, namely a "Cloudogu" logo as there may be a myriad of logos.

How about this suggestion?
```suggestion
- include Cloudogu logo in index as SVG #16 
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,15 +5,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- change gulp-file for logos to process svg-images
 
 ### Added
 - new class for danger alert with brighter background color
 - new class for success alert with brighter background color
 - new class for displaying inactive table rows
 - several new classes to display a collapsiblesr    
 - examples for the new classes in `index.html`
+- single svg-logo
```
Comment (ppxl) [inline CHANGELOG.md:21]: As above:
```suggestion
- single Cloudogu logo as SVG
```

File: src/images/logo/blib-blue-vec.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...4.14h0a67.83,67.83,0,0,1,51.64-44.64h-.06V238.61a60.68,60.68,0,1,1,24.25.15v70a68.21,68.21,0,0,1,52.27,44.61l116.53-24,.6,2.81c-.05-.93-.07-1.87-.07-2.81a60.66,60.66,0,1,1,121.31,0Z"/><path class="cls-1" d="M742,173.51c-55.32-25.88-106.5-23-147.65-6.18-2-42.67-20.88-83.76-57-118C467.11-17.18,353.86-11,287.55,62.75c-27.8,30.93-42,67.41-44.2,104.56-41.13-16.84-92.3-19.67-147.59,6.2C25.52,206.37-29.84,309.63,20.69,409.65c36.82,72.9,128.49,107.57,187,101.64l423.57.13C689.81,516.73,780.46,482,817,409.65,867.55,309.63,812.19,206.37,742,173.51Zm14.59,209.23c-21.55,39.7-58.75,54.73-100.77,58.54H506.59c-19.39,24.45-47.34,43.58-87.55,43.58-35.83,0-66.42-15.25-87.55-43.58l-148,.14c-42.66-3.58-80.52-18.49-102.34-58.68-31.84-58.66-6.58-125.45,48.37-152.32s122-1.85,148,54.67a114.28,114.28,0,0,1,6.17,18.16s4.23-5,4.93-5.77c20.46-21.57,33.53-29,53.32-39.92a91.69,91.69,0,0,1-13.78-16.36c-.57-1-1-1.81-1.23-2.12-23.16-34.32-26.71-89.82,9.24-131.84C375.52,61.2,442.68,56.93,490,95.66c31.88,26.08,52.19,79.56,27.63,131.8l-.07.1s-8.54,17.93-21.78,30c19.79,11,32.86,18.35,53.32,39.92.7.74,4.88,5.73,4.93,5.77a115.3,115.3,0,0,1,6.17-18.16c25.92-56.52,93-81.54,147.95-54.67S788.38,324.08,756.54,382.74Z"/></svg>
```
Comment (ppxl) [inline src/images/logo/blib-blue-vec.svg:1]: Is the german content in this SVG supposed to be there? Also the identifier and data name are pretty wild. I would expect both to do something with "Cloudogu" and "logo".

File: src/index.html [hunk 4]
```diff
@@ -96,6 +96,12 @@ <h3 id="logo">Logo</h3>
                 <img class="img-responsive thumbnail-primary-bg" alt="" src="images/logo/text-white-320px.png">
             </div>
         </div>
+        <!-- Vector-Logos ================================================== -->
+        <div class="row">
+            <div class="col-lg-2 bs-component">
+                <img class="img-responsive thumbnail" alt="Cloudogu Logo" src="images/logo/blib-blue-vec.svg">
```
Comment (ppxl) [inline src/index.html:102]: Good catch to add it.

Studying `index.html` I find above this usage way more PNG versions as example. Would it make sense to prefer the SVG version in any way?

One thing could be to move the vector logo above the PNG logo instances or to replace the PNGs with the SVG altogether.

**Review by ppxl — CHANGES_REQUESTED** [review]
Looks good but some points need attention

File: src/index.html (hunk 4 above)
Comment (Phil-Ah) [reply src/index.html:102]: Most images are currently unsused. I added the white logo as .svg as this sees some use as well. Further logo-images may be added as need arises

File: CHANGELOG.md (hunk 2 above)
Comment (Phil-Ah) [reply CHANGELOG.md:21]: This is changed with the last commit.

File: src/images/logo/blib-blue-vec.svg [hunk 5]
```diff
@@ -0,0 +1 @@
...
```
Comment (JBartscher) [inline src/images/logo/blib-blue-vec.svg:1]:
```suggestion
<svg id="Cloudogu_Logo_blue" data-name="Cloudogu Logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 838.67 513.77"><defs><style>.cls-1{fill:#60a1da;}</style></defs><path class="cls-1" d="M721.31,329.36a60.66,60.66,0,0,1-117.1,22.24l.53,2.51-118.1,23.37a67.73,67.73,0,0,1-135.4.17l-119-23.54a60.68,60.68,0,1,1,5.27-24.75l117.3,24.14h0a67.83,67.83,0,0,1,51.64-44.64h-.06V238.61a60.68,60.68,0,1,1,24.25.15v70a68.21,68.21,0,0,1,52.27,44.61l116.53-24,.6,2.81c-.05-.93-.07-1.87-.07-2.81a60.66,60.66,0,1,1,121.31,0Z"/><path class="cls-1" d="M742,173.51c-55.32-25.88-106.5-23-147.65-6.18-2-42.67-20.88-83.76-57-118C467.11-17.18,353.86-11,287.55,62.75c-27.8,30.93-42,67.41-44.2,104.56-41.13-16.84-92.3-19.67-147.59,6.2C25.52,206.37-29.84,309.63,20.69,409.65c36.82,72.9,128.49,107.57,187,101.64l423.57.13C689.81,516.73,780.46,482,817,409.65,867.55,309.63,812.19,206.37,742,173.51Zm14.59,209.23c-21.55,39.7-58.75,54.73-100.77,58.54H506.59c-19.39,24.45-47.34,43.58-87.55,43.58-35.83,0-66.42-15.25-87.55-43.58l-148,.14c-42.66-3.58-80.52-18.49-102.34-58.68-31.84-58.66-6.58-125.45,48.37-152.32s122-1.85,148,54.67a114.28,114.28,0,0,1,6.17,18.16s4.23-5,4.93-5.77c20.46-21.57,33.53-29,53.32-39.92a91.69,91.69,0,0,1-13.78-16.36c-.57-1-1-1.81-1.23-2.12-23.16-34.32-26.71-89.82,9.24-131.84C375.52,61.2,442.68,56.93,490,95.66c31.88,26.08,52.19,79.56,27.63,131.8l-.07.1s-8.54,17.93-21.78,30c19.79,11,32.86,18.35,53.32,39.92.7.74,4.88,5.73,4.93,5.77a115.3,115.3,0,0,1,6.17-18.16c25.92-56.52,93-81.54,147.95-54.67S788.38,324.08,756.54,382.74Z"/></svg>
```

File: src/images/logo/blib-white-vec.svg [hunk 6]
```diff
@@ -0,0 +1 @@
...24.14h0A67.83,67.83,0,0,1,408,307.71h-.06V237.46a60.68,60.68,0,1,1,24.25.15v70a68.21,68.21,0,0,1,52.27,44.61l116.53-24,.6,2.81c0-.93-.07-1.87-.07-2.81a60.66,60.66,0,1,1,121.31,0Z"/><path class="cls-1" d="M743.43,172.36c-55.32-25.88-106.5-23-147.65-6.18-2-42.67-20.88-83.76-57-118C468.59-18.33,355.34-12.19,289,61.6c-27.8,30.93-42,67.41-44.2,104.56-41.13-16.84-92.3-19.67-147.59,6.2C27,205.22-28.36,308.48,22.17,408.5c36.82,72.9,128.49,107.57,187,101.64l423.57.13c58.55,5.31,149.2-29.39,185.77-101.77C869,308.48,813.67,205.22,743.43,172.36ZM758,381.59c-21.55,39.7-58.75,54.73-100.77,58.54H508.07c-19.39,24.45-47.34,43.58-87.55,43.58-35.83,0-66.42-15.25-87.55-43.58l-148,.14c-42.66-3.58-80.52-18.49-102.34-58.68C50.81,322.93,76.07,256.14,131,229.27s122-1.85,148,54.67a114.28,114.28,0,0,1,6.17,18.16c.05,0,4.23-5,4.93-5.77,20.46-21.57,33.53-28.95,53.32-39.92a91.69,91.69,0,0,1-13.78-16.36c-.57-1-1-1.81-1.23-2.12-23.16-34.32-26.71-89.82,9.24-131.84C377,60.05,444.16,55.78,491.5,94.51c31.88,26.08,52.19,79.56,27.63,131.8l-.07.1s-8.54,17.93-21.78,30c19.79,11,32.86,18.35,53.32,39.92.7.74,4.88,5.73,4.93,5.77a115.3,115.3,0,0,1,6.17-18.16c25.92-56.52,93-81.54,147.95-54.67S789.86,322.93,758,381.59Z"/></svg>
```
Comment (JBartscher) [inline src/images/logo/blib-white-vec.svg:1]:
```suggestion
<svg id="Clodogu-Logo_white" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 838.67 513.77"><defs><style>.cls-1{fill:#fff;}</style></defs><path class="cls-1" d="M722.79,328.21a60.66,60.66,0,0,1-117.1,22.24l.53,2.51-118.1,23.37a67.73,67.73,0,0,1-135.4.17L233.74,353A60.68,60.68,0,1,1,239,328.21l117.3,24.14h0A67.83,67.83,0,0,1,408,307.71h-.06V237.46a60.68,60.68,0,1,1,24.25.15v70a68.21,68.21,0,0,1,52.27,44.61l116.53-24,.6,2.81c0-.93-.07-1.87-.07-2.81a60.66,60.66,0,1,1,121.31,0Z"/><path class="cls-1" d="M743.43,172.36c-55.32-25.88-106.5-23-147.65-6.18-2-42.67-20.88-83.76-57-118C468.59-18.33,355.34-12.19,289,61.6c-27.8,30.93-42,67.41-44.2,104.56-41.13-16.84-92.3-19.67-147.59,6.2C27,205.22-28.36,308.48,22.17,408.5c36.82,72.9,128.49,107.57,187,101.64l423.57.13c58.55,5.31,149.2-29.39,185.77-101.77C869,308.48,813.67,205.22,743.43,172.36ZM758,381.59c-21.55,39.7-58.75,54.73-100.77,58.54H508.07c-19.39,24.45-47.34,43.58-87.55,43.58-35.83,0-66.42-15.25-87.55-43.58l-148,.14c-42.66-3.58-80.52-18.49-102.34-58.68C50.81,322.93,76.07,256.14,131,229.27s122-1.85,148,54.67a114.28,114.28,0,0,1,6.17,18.16c.05,0,4.23-5,4.93-5.77,20.46-21.57,33.53-28.95,53.32-39.92a91.69,91.69,0,0,1-13.78-16.36c-.57-1-1-1.81-1.23-2.12-23.16-34.32-26.71-89.82,9.24-131.84C377,60.05,444.16,55.78,491.5,94.51c31.88,26.08,52.19,79.56,27.63,131.8l-.07.1s-8.54,17.93-21.78,30c19.79,11,32.86,18.35,53.32,39.92.7.74,4.88,5.73,4.93,5.77a115.3,115.3,0,0,1,6.17-18.16c25.92-56.52,93-81.54,147.95-54.67S789.86,322.93,758,381.59Z"/></svg>
```

File: src/images/logo/blib-blue-vec.svg (hunk 3 above)
Comment (JBartscher) [reply src/images/logo/blib-blue-vec.svg]: changed to "Cloudogu_Logo_<colorname>"

**Review by JBartscher — APPROVED** [review]
PR Ok

**Pull request merged**